orchestra/static/orchestra/team-info/team-info-card.html

```diff
@@ -10,6 +10,12 @@ <h3>
              target="_blank">
             Project Management
           </a>
+          <a class="btn"
+             ng-if="teamInfoCard.isProjectAdmin &&
+                    teamInfoCard.projectStatus != 'Aborted'"
```

Comment:
tough! we'll forget to add "or completed" when we add a completed status.

What if we explicitly check for `(projectStatus === 'Paused' || projectStatus === 'Active')`?